SignalServiceKit/src/Storage/TSStorageManager.m

```diff
@@ -541,9 +548,9 @@ - (NSData *)databasePassword
         // Try to reset app by deleting database.
         // FIXME SHARINGEXTENSION
         OWSFail(@"disabled while Share extension is WIP");
```

Comment:
Just making sure you intended to leave this `OWSFail` in. I think it will prevent new installs in the meanwhile, right?

Reply:
Removed.  Wanted to leave it in until I tested a bit.  

NOTE: I haven't tested this code path with the sharing extension yet, so don't run the sharing extension on your primary yet.